tests/output/pjs/test_utils.js

```diff
@@ -43,20 +43,16 @@ var runTest = function(options) {
     itFunc(displayTitle, function(done) {
         var output = new LiveEditorOutput({
             outputType: "pjs",
-            workersDir: "../../../build/workers/",
+            workersDir: "../../../build/",
             externalsDir: "../../../build/external/",
             imagesDir: "../../../build/images/",
             soundsDir: "../../../sounds/",
-            jshintFile: "../../../build/external/jshint/jshint.js",
             loopProtectTimeouts: {
                 initialTimeout: 1000,
                 frameTimeout: 500
             },
         });
 
-        // Switch to the Scratchpad's version
-        output.config.switchVersion(options.version);
```

Comment:
Why can we get rid of this?

Reply:
Good question. I could not find any PJS tests that specified the version option, and it's not documented, and ScratchpadConfig() isn't my favorite class to have a dependency on (it's a Backbone model), so I happily did away with it here.